media/inf/font.inf

```diff
@@ -48,20 +48,23 @@ HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","Times",0x00
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","Tms Rmn",0x00000000,"Times New Roman"
 
 [Font.Greek.Reg]
+; FIXME: I do suspect that all substitutions to "Verdana" in this section and in the corresponding Greek section in
+; base/setup/lib/muifonts.h are a hack actually. I suspect that actually "Tahoma" would be correct for all of them.
+; Using "Tahoma" looks bold erroneously. Using "Verdana" looks blurry, but (at least) not bold.
+; See https://github.com/reactos/reactos/pull/4829 for some pictures.
```

Comment:
```suggestion
; FIXME: It's suspected that all substitutions to "Verdana" in this section and in the corresponding Greek section in
; base/setup/lib/muifonts.h are a hack actually. Perhaps actually "Tahoma" would be correct for all of them.
; Using "Tahoma" looks bold erroneously. Using "Verdana" looks blurry, but (at least) not bold.
; See https://github.com/reactos/reactos/pull/4829 for some pictures.
```

Reply:
superseded by the direct push to master. The comment is no longer necessary at all.